media/css/firefox/tiles.less

```diff
@@ -4,10 +4,136 @@
 
 @import "../sandstone/lib.less";
 
+#outer-wrapper {
+    overflow: hidden;
+}
+
 #wrapper {
     margin-top: 50px;
 }
 
-.sidebar .sidebar-inset img {
-    border: none;
+.sidebar {
+    float: left;
+    margin: -200px 10px 0 10px;
+
+    .sidebar-inset {
+        background-color: transparent;
+        box-shadow: none;
+        border: none;
+        padding: 0;
+
+        img {
+            border: none;
+            max-width: none;
+            margin-bottom: @gridGutterWidth;
+        }
+
+        img.tab {
+            width: 200%;
+        }
+
+        img.tab-preferences {
+            width: 100%;
+        }
+    }
+}
+
+.main-column {
+    width: 580px;
+
+    .intro {
+        font-size: @baseLine;
+        font-size: 1.2rem;
```

Comment:
You should use the `.font-size()` mixin here, for example:

```
.font-size(@baseLine);
```

Then you do not need the two separate `font-size` entries. @see https://github.com/mozilla/bedrock/blob/master/media/css/sandstone/lib.less#L24